tests/e2e/testdata/helm-gitsync.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: numaplane.numaproj.io/v1alpha1
+kind: GitSync
+metadata:
+  name: gitsync-example
+  namespace: numaplane-system
+spec:
+  path: "sample-manifests"
+  repoUrl: http://localgitserver-service.numaplane-system.svc.cluster.local/git/repo4.git
+  targetRevision: master
+  helm: {}
```

Comment:
It will to good to add few test cases for helm value override as well.
1. Override using key/value pair
2. Override using new values.yaml file

Reply:
Test cases added to try these actions.